<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDialogClass</class>
 <widget class="QDialog" name="TestDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestDialog</string>
  </property>
  <widget class="QComboBox" name="selectTest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="testRepeat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelTest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runTest">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Run test</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="testResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save test result to file</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelTest</sender>
   <signal>clicked()</signal>
   <receiver>TestDialogClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runTest</sender>
   <signal>clicked()</signal>
   <receiver>TestDialogClass</receiver>
   <slot>runTest()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>TestDialogClass</receiver>
   <slot>saveToFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>runTest()</slot>
  <slot>getSelectedTest()</slot>
  <slot>testi()</slot>
  <slot>saveToFile()</slot>
 </slots>
</ui>
